app/models/taxon.rb

```diff
@@ -54,4 +56,10 @@ def grandchildren?
   def merge(to_merge)
     Taxon.new(content_item.merge(to_merge))
   end
+
+  def can_subscribe
+    return @can_subscribe if defined?(@can_subscribe)
```

Comment:
I guess I just find it weird that people can set this value to be things that aren't `boolean`. Because you can still do `taoxn.can_subscribe = []` for example, and then when you ask `can_subscribe?` and get an array, it's a bit weird.